app/views/student_identifier_types/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render 'organizations/organization_banner' %>
+
+<div class="site-content">
+  <div class="site-content-cap">
+    <h2 class="site-content-heading">
+      New identifier type
+    </h2>
+  </div>
+</div>
+
+<div class="site-content-body">
+  <div class="flash flash-with-icon flash-form">
+    <%= octicon 'alert' %>
+    Please do not ask students to provide any confidential information (e.g. SSN or bank account number) as their identifiers.
+  </div>
+  <%= form_for [@organization, @student_identifier_type] do |f| %>
+      <%= render partial: 'student_identifier_types/form_options', locals: { f: f } %>
+      <div class="form-actions">
+        <%= f.submit 'Create Identifier', class: 'btn btn-green js-navigation' %>
+        <%= link_to 'Cancel', session[:return_to], class: 'btn', role: 'button'%>
```

Comment:
Gotcha, also would you mind updating the button to use the Classroom green color?